lib/outline.vim

```diff
@@ -161,6 +162,8 @@ def Open(): number
     win_execute(outline_win_id, 'nnoremap <buffer> <up> <up>^')
     win_execute(outline_win_id, 'cursor(len(title) + 1, 1)')
 
+    win_execute(outline_win_id, 'w:PreWinId = ' .. win_getid())
```

Comment:
I am not super sure because it is almost time to go to sleep for me as well, but is it the same as
`win_execute(outline_win_id, 'w:PreWinId = ' .. outline_win_id)` ?

Perhaps to make it more explicit, we may use setwinvar() function?